data/json/recipes/recipe_weapon.json

```diff
@@ -1736,30 +1736,122 @@
     ]
   ]
 },{
-  "type" : "recipe",
-  "result": "acidbomb",
-  "category": "CC_WEAPON",
-  "subcategory": "CSC_WEAPON_EXPLOSIVE",
-  "skill_used": "fabrication",
-  "difficulty": 0,
-  "time": 500,
-  "reversible": true,
-  "autolearn": true,
-  "components": [
+    "type" : "recipe",
+    "result": "acidbomb_micro",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_EXPLOSIVE",
+    "skill_used": "fabrication",
+    "difficulty": 0,
+    "time": 900,
+    "reversible": true,
+    "autolearn": true,
+    "components": [
+    [
+      [ "clay_canister", 1 ],
+      [ "flask_glass", 1 ]
+    ],[
+       [ "acid", 1 ]
```

Comment:
If you're changing acid bombs, you could add some more acids here.
We have sulphuric acid (`chem_sulphuric_acid`), nitric acid (`chem_nitric_acid`) and acetic acid (`chem_acetic_acid`). They are currently quite useless, allowing them in acid bombs would give them some uses.